src/main/java/teammates/ui/webapi/CreateNotificationAction.java

```diff
@@ -0,0 +1,48 @@
+package teammates.ui.webapi;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import org.apache.http.HttpStatus;
+
+import teammates.common.datatransfer.attributes.NotificationAttributes;
+import teammates.common.exception.EntityAlreadyExistsException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Logger;
+import teammates.ui.request.CreateNotificationRequest;
+import teammates.ui.request.InvalidHttpRequestBodyException;
+
+/**
+ * Action: Creates a new notification banner.
+ */
+public class CreateNotificationAction extends AdminOnlyAction {
+    private static final Logger log = Logger.getLogger();
+
+    @Override
+    public ActionResult execute() throws InvalidHttpRequestBodyException, InvalidOperationException {
```

Comment:
```suggestion
    public JsonResult execute() throws InvalidHttpRequestBodyException, InvalidOperationException {
```
I think the return type could be more specific here.